TicTacToeApp/ComputerPlayer.swift

```diff
@@ -7,6 +7,16 @@ public class ComputerPlayer {
   }
 
   public func makeMove(gameState: Game) -> Int {
+    var move: Int?
+    let serviceUrl = "http://107.170.25.194:5000/"
+    let request = computerMoveRequest(gameState, baseUrl: serviceUrl)
+
+    makeBlockingRequest(request, onSuccess: { (data) in move = Int(data) })
+
+    return move ?? getRandomElement(availableSpaces(gameState))
```

Comment:
Given that error handling is one of the very next stories, I would remove the `getRandomElement` branch.

On top of that, assuming I had error handling, if there was a scenario that my error handling did not catch, I'd rather be notified about the error instead of having it be swallowed. Can you think of better alternative in the event of an unexpected error?